docs/data-management/metrics/funnel-metric.md

```diff
@@ -56,18 +56,39 @@ There are three options:
 ## Analysis of funnel metrics
 
 To analyze the effects of an experiment on a funnel, add it as a metric to an experiment as usual.
-Initially, the funnel will show up in a collapsed format, but can be expanded by clicking the carrot.
-
-![Funnel metric analysis](/img/data-management/metrics/funnel-analysis-collapsed.png)
 
 The results show the conversion rate from assignment to each particular step; the overall results thus are the same as the results for the last step.
 The popover shows additional details:
 
-- drop-off from the prior step indicates what fraction of users who made it to the previous step in the funnel did not make it to this step
-- survivors are the total number of users that made it all the way to this step in the funnel.
+- survivors are the total number of users that made it all the way to this step in the funnel
+- for steps after the first, the conversion rate from prior step: fraction of users who made it to the previous step in the funnel did not make it to this step
 
 ![Funnel metric analysis](/img/data-management/metrics/funnel-analysis.png)
 
+:::warning
+Comparing conversion rate from prior step across variants is fraught. The difference does _not_ show the direct effect of the treatment on that step in the funnel alone, as might be expected. Instead, it also includes the indirect effects of changing the composition of users who reach the prior step. These indirect effects are often larger than the direct effect.
+
+Because this is a common source of confusion, we do not display the lift in conversion rate from prior step nor its confidence interval. The only safe way to interpret treatment effects on funnel metrics is using the lift in conversion rate **from assignment**. This guarantees a valid comparison: at assignment, A and B groups are balanced because of randomization; after the first step, they are different and the comparison is apples-to-oranges.
```

Comment:
Is from first event ok too?

Reply:
I added "For steps beyond the first," - does that help clarify? The first step is fine because conversion from prior step and conversion from assignment coincide for it.